analyze/aps-cycle.h

```diff
@@ -4,7 +4,7 @@
  * Routines for breaking cycles involving only fibers.
  */
 
-extern void break_fiber_cycles(Declaration,STATE *);
+extern void break_fiber_cycles(Declaration,STATE *, DEPENDENCY d);
```

Comment:
Is the second argument used any more?  It seems to be assigned to a global variable and then ignored?  But maybe I missed something?